pkgs/development/python-modules/parsley/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage
+, fetchPypi
+, stdenv
```

Comment:
personal preference to avoid stdenv when possible.
```suggestion
, llib
```

Reply:
fixed